src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,41 @@
+package mate.academy.service
+
+import mate.academy.model.Circle
+import mate.academy.model.Color
+import mate.academy.model.Figure
+import mate.academy.model.FigureType
+import mate.academy.model.IsoscelesTrapezoid
+import mate.academy.model.Rectangle
+import mate.academy.model.RightTriangle
+import mate.academy.model.Square
+import kotlin.random.Random
+
+const val DEFAULT_RADIUS = 10.0
+
+class FigureSupplier {
+    private val random = Random.Default
+    private val colorSupplier = ColorSupplier()
+
+    fun getRandomFigure(): Figure {
+        val randomColor = colorSupplier.getRandomColor()
+        val randomFigureType = FigureType.values()[Random.nextInt(FigureType.values().size)]
+
+        return when(randomFigureType) {
+            FigureType.CIRCLE
```

Comment:
For readability improving Figure's entities could be imported by using Alt+Enter hotkey and "add import for..."

Not a mistake just FYI